bazel/foreign_cc/luajit.patch

```diff
@@ -102,6 +99,17 @@ index 00000000..3b4f8eca
 +    export TARGET_CFLAGS="${TARGET_CFLAGS} -fsanitize-blacklist=${BLOCK_PATH}"
 +    echo "fun:*" > clang-asan-blocklist.txt
 +fi
++if [[ "$(uname -s)" == "Linux" ]]; then
```

Comment:
nit-suggestion: assuming the code above doesn't change/use the CFLAGS/LDFLAGS, consider adding this if-else right after `export TARGET_CFLAGS` above (line 90) as it is logically coupled with that code.